src/transport/raw/TCP.cpp

```diff
@@ -309,29 +303,25 @@ CHIP_ERROR TCPBase::ProcessReceivedBuffer(Inet::TCPEndPoint * endPoint, const Pe
             return err;
```

Comment:
This is returning a CHIP_ERROR as an INET_ERROR.  That doesn't seem right.

And above, where we return CHIP_NO_ERROR we should have INET_NO_ERROR, right?

Reply:
Yeah, solving this in C++ without making all the error types struct wrappers for integers is a PITA... :(